src/components/VerticalListMenu/VerticalListMenu.spec.jsx

```diff
@@ -0,0 +1,160 @@
+import _ from 'lodash';
```

Comment:
Many of our tests are organized like
```
component {
  render {...}
  props {...}
  childComponents {...}
}
``` but this file kind of breaks that pattern.